src/python/WMCore/Database/CMSCouch.py

```diff
@@ -346,6 +347,39 @@ def updateDocument(self, doc_id, design, update_func, fields={}, useBody=False):
                         (self.name, design, update_func, doc_id)
             return self.put(uri=updateUri, data=fields, decode=False)
 
+    def updateBulkDocuments(self, doc_ids, paramsToUpdate, updateLimits=1000):
+
+        uri = '/%s/_bulk_docs/' % self.name
+        conflictDocIDs = []
+        for ids in grouper(doc_ids, updateLimits):
+            # get original documens
+            docs = self.allDocs(options={"include_docs": True}, keys=ids)['rows']
+            data = {}
+            data['docs'] = []
+            for j in docs:
+                doc = {}
+                doc.update(j['doc'])
+                doc.update(paramsToUpdate)
```

Comment:
I guess we don't have inner dicts to be updated, do we?

Reply:
I don't think there is nested dict to update in WQ